tests/lean/interactive/completion7.lean.expected.out

```diff
@@ -15,6 +15,10 @@
     "`and x y`, or `x && y`, is the boolean \"and\" operation (not to be confused\nwith `And : Prop → Prop → Prop`, which is the propositional connective).\nIt is `@[macro_inline]` because it has C-like short-circuiting behavior:\nif `x` is false then `y` is not evaluated.\n",
     "kind": "markdown"},
    "detail": "Bool → Bool → Bool"},
+  {"label": "false_and",
+   "kind": 3,
+   "detail": "∀ (p : Prop), (False ∧ p) = False"},
+  {"label": "true_and", "kind": 3, "detail": "∀ (p : Prop), (True ∧ p) = p"},
   {"label": "AndOp",
```

Comment:
@rish987 Sorry for noticing only now, but does this order suggest that the end-of-word bonus should maybe be smaller than beginning-of-word?

Reply:
Yes, thanks for catching this. Fixed!